docs/TODO.md

```diff
@@ -0,0 +1,35 @@
+# TODO
```

Comment:
db can come as the lowest priority. Most of state should be persistent on-chain until we have p2p connections for verifiers.

Reply:
Get it